aw_datastore/storages/peewee.py

```diff
@@ -0,0 +1,113 @@
+from typing import Optional, List
+from datetime import datetime
+import json
+import os
+
+from peewee import Model, CharField, DateTimeField
+from playhouse.sqlite_ext import SqliteExtDatabase
+
+from aw_core.models import Event
+
+from . import logger, data_dir
+from .abstract import AbstractStorage
+
+# TODO: Make dependent on testing variable in constructor
+db = SqliteExtDatabase(os.path.join(data_dir, 'peewee-sqlite.db'))
+
+logger = logger.getChild("peewee")
+
+
+class BaseModel(Model):
+    class Meta:
+        database = db
+
+
+class BucketModel(BaseModel):
+    id = CharField(unique=True)
```

Comment:
No, that would make the column name `bid` as well, and I'd prefer it if the table's primary key remains to be called `id`. I kind of agree with this sentiment, especially when it comes to class members: http://stackoverflow.com/posts/77925/revisions